chia/full_node/full_node_api.py

```diff
@@ -201,47 +201,9 @@ async def new_transaction(
             new_set = set()
             new_set.add(peer.peer_node_id)
             self.full_node.full_node_store.peers_with_tx[transaction.transaction_id] = new_set
-
-            async def tx_request_and_timeout(full_node: FullNode, transaction_id: bytes32, task_id: bytes32) -> None:
```

Comment:
I think this function was in this file because it's part of the `_api` functionality. I think it would make more sense to move it out to be a free function, but still in the `full_node_api.py` file.